pkg/virtual/workspaces/registry/rest.go

```diff
@@ -154,6 +157,21 @@ func (s *REST) NamespaceScoped() bool {
 	return false
 }
 
+func (s *REST) isUserAllowed(user user.Info, name string, reviewer workspaceauth.Reviewer) (bool, error) {
```

Comment:
what does allowed mean? hasUserWorkspaceAccess ?

Reply:
what is name? userName? workspaceName? needs a better variable name.

Reply:
also please qualify reviewer. Is that the org workspace reviewer?

Reply:
name is the review `resourceName`, generally, so here obviously the workspace Name
let's simply call this method `review`  and change `name` to `workspaceName` ?